go-for-eat/middleware/api.js

```diff
@@ -1,69 +1,67 @@
-// import { normalize } from 'normalizr';
-const baseUrl = '';
+import { normalize } from 'normalizr';
+import baseUrl from '../config/serverHost.js';
 
 
-const callApi = (endpoint, method='GET', body, accessToken) => {
+const callApi = (endpoint, method='GET', body, accessToken, schema) => {
   const fullUrl = baseUrl + endpoint;
-
-
   const headers = {};
-  if (accessToken) headers.Authorization = `Bearer ${accessToken}`;
+  if (accessToken) headers.authorization = `Bearer ${accessToken}`;
   if (method === 'POST' || method === 'PUT') headers['Content-Type'] = 'application/json';
 
   return fetch(fullUrl, {
     method,
     headers,
     body
   })
-    .then(response => response.json())
-
-}
+    .then(response => {return response._bodyInit ?  response.json() : null;})
```

Comment:
response => response._bodyInit ? response.json() : null. Why did you need the curly braces if you are going to make it a one-liner anyway?